File: Java/src/Stove.java
public class Stove {
	private Pan pan;
	public boolean state = false;

	public void Pan(Pan p) {
		pan = p;
	}
	
	public Pan getPan(){
		return pan;
	}

	public void Cook() {
		if (state) {
			while (pan.Is_ready()) {
				pan.GetHeat();
			}
		}
	}
}

File: labaTP1/WindowsFormsApplication3/Stove.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsFormsApplication3
{
    class Stove
    {
        private Pan pan;
        public bool state { set; get; }
        public Pan Pan { set { pan = value; }get { return pan; } }

        public void Cook()
        {
            if (state)
            {
                while (pan.Is_ready())
                {
                    pan.GetHeat();
                }
            }
        }
    }
}
